eng/Version.Details.xml

```diff
@@ -1,78 +1,78 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Dependencies>
-  <Source Uri="https://github.com/dotnet/dotnet" Mapping="wpf" Sha="54c879632fdf0197d517d80e4b90861e2bd5d03e" BarId="289655" />
+  <Source Uri="https://github.com/dotnet/dotnet" Mapping="wpf" Sha="77ee357638bcd8fa66a1c16fa588dcd5818068d2" BarId="289869" />
   <ProductDependencies>
-    <Dependency Name="Microsoft.Private.Winforms" Version="10.0.0-rc.1.25555.103">
+    <Dependency Name="Microsoft.Private.Winforms" Version="10.0.0-rc.3.25556.111">
       <Uri>https://github.com/dotnet/dotnet</Uri>
-      <Sha>54c879632fdf0197d517d80e4b90861e2bd5d03e</Sha>
+      <Sha>77ee357638bcd8fa66a1c16fa588dcd5818068d2</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Dotnet.WinForms.ProjectTemplates" Version="10.0.0-rc.1.25555.103">
+    <Dependency Name="Microsoft.Dotnet.WinForms.ProjectTemplates" Version="10.0.0-rc.3.25556.111">
       <Uri>https://github.com/dotnet/dotnet</Uri>
-      <Sha>54c879632fdf0197d517d80e4b90861e2bd5d03e</Sha>
+      <Sha>77ee357638bcd8fa66a1c16fa588dcd5818068d2</Sha>
     </Dependency>
-    <Dependency Name="System.Reflection.MetadataLoadContext" Version="10.0.0-rc.1.25555.103">
+    <Dependency Name="System.Reflection.MetadataLoadContext" Version="10.0.0-rc.3.25556.111">
       <Uri>https://github.com/dotnet/dotnet</Uri>
-      <Sha>54c879632fdf0197d517d80e4b90861e2bd5d03e</Sha>
+      <Sha>77ee357638bcd8fa66a1c16fa588dcd5818068d2</Sha>
     </Dependency>
-    <Dependency Name="System.Windows.Extensions" Version="10.0.0-rc.1.25555.103">
+    <Dependency Name="System.Windows.Extensions" Version="10.0.0-rc.3.25556.111">
       <Uri>https://github.com/dotnet/dotnet</Uri>
-      <Sha>54c879632fdf0197d517d80e4b90861e2bd5d03e</Sha>
+      <Sha>77ee357638bcd8fa66a1c16fa588dcd5818068d2</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.NETCore.Platforms" Version="10.0.0-rc.1.25555.103">
+    <Dependency Name="Microsoft.NETCore.Platforms" Version="10.0.0-rc.3.25556.111">
       <Uri>https://github.com/dotnet/dotnet</Uri>
-      <Sha>54c879632fdf0197d517d80e4b90861e2bd5d03e</Sha>
+      <Sha>77ee357638bcd8fa66a1c16fa588dcd5818068d2</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.DotNet.Wpf.DncEng" Version="10.0.0-rc.1.25552.1">
+    <Dependency Name="Microsoft.DotNet.Wpf.DncEng" Version="10.0.0-rc.1.25513.8">
```

Comment:
Looks like this dependency now gets updated because of this line when building wpf inside the VMR internally: https://github.com/dotnet/wpf/blob/42f7d5d89b211409b9c11a294bfcd78cbe748664/eng/WpfArcadeSdk/tools/Packaging.props#L48-L49

@mmitche for a second pair of eyes.

I'm not sure why this only shows up now but if I'm reading things correctly this races between the internal VMR build and the internal wpf-int build.